pyproject.toml

```
@@ -16,7 +16,7 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = "^3.7"
-graphene = "~=3.1"
+graphene = "~=3.0b8"
```

older betas have `graphql-core>=3.1.2,<4` which is dangerous to support. from here on theres an upper pin on the minor. I hope that going forth, graphene keep maintaining a 'proper' upper pin, and this library can follow suit whenever needed.

with regard to the latter: could consider adding `graphene<3.2` here, but decided against it as here's not really a dependency on the graphene code base as much as on the graphql-core codebase... could also consider adding an upper pin on graphql-core here.

---

so to go one step stricter in avoiding compatibility issues, and in order to leave the original graphene lower pin untouched:

- option 1:
  ```suggestion
  graphene = ">=3.0b6"
  graphql-core = ">=3.1,<3.3"
  ```
- option 2 (even more strict, basically avoiding the same mistake graphene made in earlier betas):
  ```suggestion
  graphene = ">=3.0b6,<3.2"
  graphql-core = ">=3.1,<3.3"
  ```

---

> but decided against it as here's not really a dependency on the graphene code base as much as on the graphql-core codebase

option 3 another safe way to avoid having to pin `graphql-core` in this lib:
```suggestion
graphene = ">=3.0b8,<3.2"
```
knowing that this graphene range has that pin on graphql-core

---

I tend towards option 1:

- option 1 will most likely have the longest support for newer graphene versions without introducing breaking graphql-core changes
- options 2,3 will probably need versioning PRs on this library in the short term